examples/references.bib

```diff
@@ -302,6 +358,13 @@ @misc{szegedy2014going
   archiveprefix = {arXiv},
   primaryclass  = {cs.CV}
 }
+@online{vandenbergSPSS,
+  author        = {van den Berg, R. G},
+  title         = {SPSS Moderation Regression Tutorial},
+  year          = XXXXX,
```

Comment:
I think this is breaking the docs build. Can you try removing the year key altogether?